Sources/Sentry/SentryCoreDataTrackingIntegration.m

```diff
@@ -0,0 +1,54 @@
+#import "SentryCoreDataTrackingIntegration.h"
+#import "SentryCoreDataSwizzling.h"
+#import "SentryCoreDataTracker.h"
+#import "SentryLog.h"
+#import "SentryNSDataSwizzling.h"
+#import "SentryOptions+Private.h"
+#import "SentryOptions.h"
+
+@interface
+SentryCoreDataTrackingIntegration ()
+
+@property (nonatomic, strong) SentryCoreDataTracker *tracker;
+
+@end
+
+@implementation SentryCoreDataTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
+    if (!options.enableAutoPerformanceTracking) {
+        [SentryLog logWithMessage:@"Not going to enable NetworkTracking because "
+                                  @"enableAutoPerformanceTracking is disabled."
```

Comment:
```suggestion
    if (!options.enableAutoPerformanceTracking) {
        [SentryLog logWithMessage:@"Not going to enable CoreData performance tracking because "
                                  @"enableAutoPerformanceTracking is disabled."
```